backend/inspirehep/files/api/s3.py

```diff
@@ -99,7 +99,21 @@ def get_file_url(key):
 
     @staticmethod
     def get_content_disposition(filename):
-        return f'attachment; filename="{secure_filename(filename)}"'
+        without_subformat = S3.remove_subformat(filename)
```

Comment:
maybe ```filename_without_subformat```?

Reply:
i think, one understands that it's a file name, while reading this line.

Reply:
It is understandable although I think ```filename_without_subformat``` is clearer and more understandable. I don't have a strong opinion about it, keep it like this